### ISSUE TITLE
Can't open gist

### ISSUE THREAD

**Issue by tedbeer** [condemil/Gist#129]
Environment: Win7, ST3
The token I created has "public_repo" and "gist" scopes.
Changes in configuration:
- "token"
- "include_users" has my user name
- "show_authors": true,

Tools -> Gist -> Open Gist fails:

```
Traceback (most recent call last):
  File "gist in D:\Users\ebespalov\AppData\Roaming\Sublime Text 3\Installed Packages\Gist.sublime-package", line 105, in _fn
  File "gist in D:\Users\ebespalov\AppData\Roaming\Sublime Text 3\Installed Packages\Gist.sublime-package", line 625, in run
  File "gist in D:\Users\ebespalov\AppData\Roaming\Sublime Text 3\Installed Packages\Gist.sublime-package", line 332, in gists_filter
  File "gist in D:\Users\ebespalov\AppData\Roaming\Sublime Text 3\Installed Packages\Gist.sublime-package", line 313, in gist_title
AttributeError: 'NoneType' object has no attribute 'get'
error: Gist: unknown error (please, report a bug!)
```

**Comment by pattulus** [comment]
@willkelleher Thanks. Now I can at least use this feature.

**Comment by shao362** [comment]
I have the same issue today and I found it's caused by module name collision with CTag package in my case.

Replacing "import helpers" with "from Gist import helpers" in gist.py of Gist.sublime-package seems worked for me.

**Comment by condemil** [comment]
@shao362 Thanks for reporting about module collision, I renamed the modules to decrease the probability of further collisions.

**Comment by condemil** [comment]
@tedbeer @willkelleher @pattulus Your issue should be fixed by now with the following commit: 52f3c187f5874be6c960caee7390448e02ddb9b8
